flink-table-store-format/src/main/java/org/apache/flink/table/store/format/orc/OrcWriterFactory.java

```diff
@@ -108,6 +128,11 @@ public FormatWriter create(PositionOutputStream out) throws IOException {
         return new OrcBulkWriter(vectorizer, new WriterImpl(null, unusedPath, opts));
     }
 
+    @Override
+    public void setLevel(int level) {
```

Comment:
Maybe we can introduce `FormatWriter create(PositionOutputStream out, String compression) throws IOException` for `FormatWriterFactory`.

Reply:
ok,I will update it

Reply:
I update it to `create(PositionOutputStream out, int level)` ,  because we can only get the `level`, and we also need to get the compression value by the `level`